Makefile

```diff
@@ -14,7 +16,8 @@ TEMPORARY_INSTALLER_FOLDER=$(TEMPORARY_FOLDER)/install
 XCODEBUILD_DERIVED_DATA=$(TEMPORARY_FOLDER)/xcodebuild/DerivedData/MockingbirdFramework
 XCODE_PATH=$(shell xcode-select --print-path)
 CLI_BUNDLE_PLIST=Sources/MockingbirdCli/Info.plist
-VERSION_STRING?=$(shell /usr/libexec/PlistBuddy -c "Print :CFBundleShortVersionString" "$(CLI_BUNDLE_PLIST)")
+VERSION?=$(shell /usr/libexec/PlistBuddy -c "Print :CFBundleShortVersionString" "$(CLI_BUNDLE_PLIST)")
```

Comment:
wonder if we should namespace this like MOCKINGBIRD_VERSION or something

Reply:
`VERSION` -> `MKB_VERSION`